### PR TITLE
#7 implement user model

### PR DESCRIPTION
**Pull request by Sontsevolosa**
* Adding 'devise' gem and configurations for it
* Implement User model
* Adding first_name and last_name columns to the users table

@mluts please, review my code

### CODE REVIEW

**Comment by mluts** [comment]
@Sontsevolosa Please write invitation for review in separate comment

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+  # confirmation, reset password and unlock tokens in the database.
+  # Devise will use the `secret_key_base` on Rails 4+ applications as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  # config.secret_key = 'd52600bdb10de2766e6b1a4631ddc9da87c91b2ff4b9dfa1326a1c1431e1bf962d70f52639bd08de443335c7c721908fc1a8eb32fc2ff2a9b607f35ff1ad3f26'
+
+  # ==> Mailer Configuration
+  # Configure the e-mail address which will be shown in Devise::Mailer,
+  # note that it will be overwritten if you use your own mailer class
+  # with default "from" parameter.
+  config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
```
Comment (mluts) [inline config/initializers/devise.rb]: @Sontsevolosa could you fill in this configuration field ?

Comment (mluts) [inline config/initializers/devise.rb]: Example
```
config.mailer_sender = 'Lunches <contact@lunches.com>'
```

File: config/routes.rb [hunk 2]
```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  devise_for :users
```
Comment (mluts) [inline config/routes.rb]: @Sontsevolosa Please add newline here to have visual separation of two distinct code parts

**Comment by mluts** [comment]
@Sontsevolosa Please consider my comments

File: config/initializers/devise.rb (hunk 1 above)
Comment (Sontsevolosa) [inline config/initializers/devise.rb]: @mluts  thank you for the example - at first I didn't understand what did you mean :)

**Comment by Sontsevolosa** [comment]
@mluts please, review my changes

**Comment by mluts** [comment]
@Sontsevolosa good, please merge

**Pull request merged**